plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/FileSystemExchange.java

```diff
@@ -65,6 +65,8 @@
         implements Exchange
 {
     private static final Pattern PARTITION_FILE_NAME_PATTERN = Pattern.compile("(\\d+)_(\\d+)\\.data");
+    private static final char[] ALPHANUMERIC = "abcdefghijklmnopqrstuvwzyz0123456789".toCharArray();
```

Comment:
`ALPHANUMERIC` usually implies `[A-Za-z0-9]`. How about calling it `RANDOMIZED_PREFIX_ALPHABET`?